src/modules/home/components/LoggedInHome/LoggedInHome.tsx

```diff
@@ -45,6 +46,7 @@ const LoggedInHome: FC = () => {
 	const [query, setQuery] = useQueryParams({
 		[VISITOR_SPACE_SLUG_QUERY_KEY]: StringParam,
 		[SEARCH_QUERY_KEY]: StringParam,
+		[SCROLL_TO_SECTION]: StringParam,
```

Comment:
https://v5.reactrouter.com/web/api/Hooks/uselocation lijkt me een betere match

Reply:
@ian-emsens-shd met useLocation kan ik toch enkel het huidige/vorige pad opvragen?
Hoe kan ik daarmee scrollen naar een specifieke sectie in de pagina?

Reply:
ok, fixed with next router?.asPath?.split?.('#')?.[0]
aangezien we geen react-router gebruiken